src/haxelib/client/Main.hx

```diff
@@ -48,6 +48,16 @@ private enum CommandCategory {
 	Deprecated(msg:String);
 }
 
+private typedef LibInstallationInfo = {
+	name:String,
+	version:String,
+	type:String,
```

Comment:
At this point it would probably be best to change `type` into an enum, with cases `Dev(path:String)`, `Git(url:String, branch:String, subDir:String)`, `Haxelib`.

Reply:
Yeah there's many things that can be rewritten.. But are we doing it?

Reply:
🤷‍♂ yeah there are. Rewriting this part in particular in this PR seems like a convenient opportunity though.

Reply:
I knew I couldn't stop if I started.. Damn I don't know where this will end x)